switch-configuration/README.md

```diff
@@ -68,16 +75,85 @@ RSRVD	<number_of_ports>
 VLAN	<vlan_name> <number_of_ports> [<ip6_address> [<ip4_address>]]
 TRUNK	<port> <vlan_name>[,<vlan_name>...]
 JUNOS	<junos_version>
-
+```
 
 
 # Scripts (No User Serviceable Parts inside)
+scripts/
+
+# Standard Operational Procedures
+## How to build a set of switch configurations
+Once the configuration files are all set up (as described above) and you have set up
+authentication parameters as described below, simply run
+```
+scripts/buildswitches
```

Comment:
Whatever this script might end up being I'd like it to have a deliberate flag for doing a rolling update of all the switches